Rusu Sabina/index.css

```diff
@@ -0,0 +1,13 @@
+h1 {
+text-align: center;
+color: blue;
```

Comment:
Schimba, te rog, culoarea lui h1 din `blue` in `#1c00a8`

Reply:
Am schimbat, sper că acum e ok.